frontend/public/components/_sysevent-stream.scss

```diff
@@ -91,8 +91,13 @@ $color-dark-border: #ddd;
   justify-content: space-between;
 }
 
+.co-sysevent__namespacelink {
+  display: none !important;
```

Comment:
We should just use classes like `hidden-xs` to hide the link at mobile.

https://getbootstrap.com/docs/3.3/css/#responsive-utilities

If we do it this way, I don't think we should need `!important`